lib/level_web/controllers/accept_invitation_controller.ex

```diff
@@ -13,14 +14,14 @@ defmodule LevelWeb.AcceptInvitationController do
     case Spaces.accept_invitation(invitation, user_params) do
       {:ok, %{user: user}} ->
         conn
-        |> LevelWeb.Auth.sign_in(invitation.space, user)
+        |> Auth.sign_in(invitation.space, user)
         |> redirect(to: cockpit_path(conn, :index))
 
       {:error, :user, changeset, _} ->
         conn
         |> assign(:changeset, changeset)
         |> assign(:invitation, invitation)
-        |> render(LevelWeb.InvitationView, "show.html")
+        |> render(InvitationView, "show.html", [])
```

Comment:
This was one bug that dialyzer caught. `render/4` needs the assigns as the fourth argument, so this would have raised an error otherwise.